### PR TITLE
Homescreen submission & review

### PR DESCRIPTION
**Pull request by washort**

### CODE REVIEW

File: mkt/constants/applications.py [hunk 1]
```diff
@@ -58,3 +58,5 @@ def get_device(request):
 
 def get_device_id(request):
     return getattr(get_device(request), 'id', None)
+
+HOMESCREEN = 'homescreen'
```
Comment (diox) [inline mkt/constants/applications.py]: Is that the tag name ? Weird place to put it, this file only contains devices info.

File: mkt/webapps/indexers.py [hunk 2]
```diff
@@ -297,6 +297,8 @@ def extract_document(cls, pk=None, obj=None):
         d['is_rereviewed'] = is_rereviewed
         d['rereview_date'] = (obj.rereviewqueue_set.get().created
                               if is_rereviewed else None)
+        is_homescreen = obj.tags.filter(tag_text=mkt.HOMESCREEN).exists()
+        d['is_homescreen'] = is_homescreen
```
Comment (diox) [inline mkt/webapps/indexers.py]: Would be nice to start adding tests to `mkt/webapps/tests_indexers.py` - it does not cover much right now, but the more we add the quicker coverage will improve :)

File: mkt/reviewers/utils.py [hunk 3]
```diff
@@ -682,6 +684,28 @@ def get_pending_queue(self):
             addon__disabled_by_user=False,
             addon__status=mkt.STATUS_PENDING)
             .exclude(addon__id__in=self.excluded_ids)
+            .exclude(addon__tags__tag_text='homescreen')
```
Comment (diox) [inline mkt/reviewers/utils.py]: Do we need to do that in other queues as well ? In what queues should updated homescreens end up ?

Comment (washort) [inline mkt/reviewers/utils.py]: I don't know the answer to that one.

Comment (washort) [inline mkt/reviewers/utils.py]: Updated homescreens now go in homescreen queue; re-reviews/escalations for homescreens go in app re-review/escalation queues.

**Pull request merged**